hibernate-core/src/main/java/org/hibernate/sql/ast/SqlTreePrinter.java

```diff
@@ -215,7 +215,7 @@ private void logNode(String text, Runnable subTreeHandler, boolean indentContinu
 
 	private void logWithIndentation(Object line) {
 		pad( depth );
-		buffer.append( line ).append( '\n' );
+		buffer.append( line ).append( String.format( "%n" ) );
 	}
```

Comment:
Given that we need to support SQL Server, it might be a good idea to use `%n` instead of `\n` to accommodate Windows OS or others.